File: frontend/src/app/pages/titulo/model/titulo.ts
import {Classe} from '../../classe/model/classe';
import {Diretor} from '../../diretor/model/diretor';
import {Categoria} from './categoria';

export class Titulo {
  id: number;
  nome: string;
  ano: Date;
  sinopse: string;
  categoria: Categoria;
  diretor: Diretor;
  classe: Classe;
}

File: desktop-frontend/src/main/java/domain/Titulo.java
package domain;

import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;
import java.util.List;

@Getter
@Setter
public class Titulo implements Serializable {

    private Long id;
    private String nome;
    private String ano;
    private String sinopse;
    private Categoria categoria;
    private Diretor diretor;
    private Classe classe;
    private List<Ator> atores;

}
